engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterSceneDelegate.m

```diff
@@ -6,12 +6,25 @@
 
 #import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate_Internal.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterSceneLifecycle.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterSharedApplication.h"
 
 FLUTTER_ASSERT_ARC
 
+@interface FlutterSceneDelegate () <FlutterSceneLifeCycleProvider>
```

Comment:
`FlutterSceneLifeCycleProvider` is a private protocol for now, but will later become public. It's purpose is to allow add-to-app developers who can't adopt the FlutterSceneDelegate to still be able to use plugin scene lifecycle events. 

See app equivalent for example of future use case: https://docs.flutter.dev/add-to-app/ios/add-flutter-screen#if-you-cant-directly-make-flutterappdelegate-a-subclass